spiceaidocs/docs/components/data-connectors/odbc.md

```diff
@@ -206,3 +240,25 @@ sql> select * from spice_test;
 
 Query took: 1.8504053329999999 seconds. 3/3 rows displayed.
 ```
+
+## Examples
```

Comment:
We should have a Postgres example too, probably one that matches the quickstart.

Reply:
Done